test/lib/code_corps/stripe_service/stripe_platform_customer_service_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule CodeCorps.StripeService.StripePlatformCustomerServiceTest do
+  use CodeCorps.ModelCase
+
+  alias CodeCorps.StripeService.StripePlatformCustomerService
+
+  describe "update/2" do
+    test "performs update" do
+      customer = insert(:stripe_platform_customer)
+      {:ok, customer} = StripePlatformCustomerService.update(customer, %{email: "mail@mail.com"})
+      assert customer.email == "mail@mail.com"
+
+       # TODO: Figure out testing if stripe API request was made
```

Comment:
It's intentional. I'm not sure how to check if a stripe API update was made. Maybe the function should return both the updated local record and the updated stripe record?